xml/System.IO/FileShare.xml

```diff
@@ -61,7 +61,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>Contains constants for controlling the kind of access other <see cref="T:System.IO.FileStream" /> objects can have to the same file.</summary>
+    <summary>Contains constants for controlling the kind of access other code can have to the same file.</summary>
```

Comment:
Maybe "clients", "callers", "consumers"?  It's any other caller to the CreateFile API, either in process or out.  Saying "code" might imply too limited a scope.  cc @dotnet/area-system-io

Reply:
Perhaps this is overly pedantic, but I suspect it's not just CreateFile, but NtXXXXX or ZwXXXX or whatever. @JeremyKuhne 

Open to any.

Reply:
Here's an example of where the CreateFileW docs use "process", which while correct, is not complete, as it's surely any other handle, except perhaps a duplicated one (on Unix, flock definitely applies to a file descriptor)
```
Enables subsequent open operations on a file or device to request read access.
Otherwise, no process can open the file or device if it requests read access.
```

Reply:
perhaps ".. kind of access other code not using this FileStream can have .." is accurate and complete enough.

Reply:
"users" maybe? I'm fine with whatever- it helps encourage broader thinking that end users will be affected using "normal" OS interaction. At least on Windows it's a low-level construct- there is no way around it.

Reply:
right, although of course it could be the very next line of code, running as the same user on the same thread, that tries to open the file to write..

Reply:
I'd go with `calls`, `callers`, or `operations`, personally.  The last one (`operations`) has the benefit of matching the terminology of the underlying documentation of our original platform.